src/khoj/interface/web/agent.html

```diff
@@ -0,0 +1,286 @@
+        #agent-avatar {
+            width: 50px;
+            height: 50px;
+            border-radius: 50%;
+            object-fit: cover;
+            margin-right: 10px;
+        }
+
+        #agent-name {
+            font-size: 18px;
+            font-weight: bold;
+            color: #333;
+        }
+
+        #agent-tuning, #agent-public, #agent-creator, #agent-managed-by-admin {
+            font-size: 14px;
+            color: #666;
+        }
+
+        #agent-tuning p {
+            white-space: pre-line;
+        }
+
+        #agent-metadata p {
+            margin: 0;
+            padding: 0;
+        }
+
+        #agent-public {
+            display: grid;
+            grid-template-columns: auto 1fr;
+            grid-gap: 12px;
+        }
+
+        .switch {
+            position: relative;
+            display: inline-block;
+            width: 50px;
+            height: 24px;
+        }
+
+        .switch input {
+            opacity: 0;
+            width: 0;
+            height: 0;
+        }
+
+        .slider {
+            position: absolute;
+            cursor: pointer;
+            top: 0;
+            left: 0;
+            right: 0;
+            bottom: 0;
+            background-color: #ccc;
+            -webkit-transition: .4s;
+            transition: .4s;
+        }
+
+        .slider:before {
+            position: absolute;
+            content: "";
+            height: 16px;
+            width: 16px;
+            left: 4px;
+            bottom: 4px;
+            background-color: white;
+            -webkit-transition: .4s;
+            transition: .4s;
+        }
+
+        input:checked + .slider {
+            background-color: var(--primary-hover);
+        }
+
+        input:focus + .slider {
+            box-shadow: 0 0 1px var(--primary-hover);
+        }
+
+        input:checked + .slider:before {
+            -webkit-transform: translateX(26px);
+            -ms-transform: translateX(26px);
+            transform: translateX(26px);
+        }
+
+        div#agent-data-wrapper {
+            display: grid;
+            grid-template-columns: 1fr;
+            grid-gap: 10px;
+            text-align: left;
+        }
+
+        /* Rounded sliders */
+        .slider.round {
+            border-radius: 34px;
+        }
+
+        .slider.round:before {
+            border-radius: 50%;
+        }
+
+        @media only screen and (min-width: 700px) {
+            body {
+                grid-template-columns: auto min(70vw, 100%) auto;
+            }
+            body > * {
+                grid-column: 2;
+            }
+            #agent-metadata-wrapper {
+                display: block;
+                width: min(30vw, 100%);
+                margin-left: auto;
+                margin-right: auto;
+            }
+        }
+    </style>
+    <script>
+        async function openChat(agentId) {
+            let response = await fetch(`/api/chat/sessions?agent_slug=${agentId}`, { method: "POST" });
```

Comment:
nit: s/agentId/agentSlug/

```suggestion
        async function openChat(agentSlug) {
            let response = await fetch(`/api/chat/sessions?agent_slug=${agentSlug}`, { method: "POST" });
```